app/assets/javascripts/two-pane.js

```diff
@@ -7,7 +7,7 @@ $.fn.tab = function ( option ) {
   }
 }
 
-$(document).on('click', ".table-two-pane td a[href$='edit']", function(e) {
+$(document).on('click', ".table-two-pane td[class$='display-two-pane'] a", function(e) {
```

Comment:
leftovers - at first it was called two-pane and other elements also had a class with the same name
I'll change it